arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -74,7 +79,7 @@
             "description": "The requested automatic tuning resource state.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/AutomaticTuningUpdate"
+              "$ref": "#/definitions/AutomaticTuning"
```

Comment:
this model is defined as a list, is that appropriate? the model is defined as a pageable model